discord/ext/commands/errors.py

```diff
@@ -1080,7 +1080,7 @@ class ExtensionNotFound(ExtensionError):
     """
 
     def __init__(self, name: str) -> None:
-        msg = f'Extension {name!r} could not be loaded.'
+        msg = f'Extension {name!r} could not be found at the specified location.'
```

Comment:
```suggestion
        msg = f'Extension {name!r} could not be loaded or found.'
```

This wording might be better?

Reply:
Fine with me!